tests/Shared/RequiresDockerTheoryAttribute.cs

```diff
@@ -3,12 +3,15 @@
 
 using Xunit;
 
-namespace Aspire.RabbitMQ.Client.Tests;
+namespace Aspire.InternalTesting;
 
 // TODO: remove these attributes when Helix has a Windows agent with Docker support
 public class RequiresDockerTheoryAttribute : TheoryAttribute
 {
-    public static bool IsSupported => !OperatingSystem.IsWindows() || Environment.GetEnvironmentVariable("HELIX_WORKITEM_ROOT") is null;
+    // Not available on windows/CI
+    public static bool IsSupported => !OperatingSystem.IsWindows() ||
+                                      (Environment.GetEnvironmentVariable("HELIX_WORKITEM_ROOT") is null
+                                      && Environment.GetEnvironmentVariable("BUILD_BUILDID") is null);
```

Comment:
Why do we need to add `BUILD_BUILDID`? 

I assume you are trying to support the case where these tests try to run on the azdo build pipeline machines, but does Docker work on the azdo pipeline machines on Linux?

Reply:
This was to make sure that we don't run any `RequiresDocker*` tests on windows/CI at all - helix or build machines.